loader/loader.go

```diff
@@ -262,7 +304,7 @@ func Load(configDetails types.ConfigDetails, options ...func(*Options)) (*types.
 	}
 
 	if opts.ResolvePaths {
-		err = ResolveRelativePaths(project)
+		err := ResolveRelativePaths(project)
 		if err != nil {
 			return nil, err
 		}
```

Comment:
nit: could also be apply to the following `if !opts.` blocks
```suggestion
		if err := ResolveRelativePaths(project); err != nil {
			return nil, err
		}
```

Reply:
I personally consider those to have lower readability for the tiny benefit of a single line being saved

Reply:
As you wish, I don't have a strong opinion about this 😉